bcrs-ui/src/main.ts

```diff
@@ -0,0 +1,37 @@
+// Vue Libraries
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import router from './router'
+import { store } from '@/store'
+
+// Styles
+import '@/assets/styles/base.scss'
```

Comment:
Also add this:
`import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-loader`

Reply:
Done, and added dependency 👍